noita_mod/core/files/append/perk_pickup.lua

```diff
@@ -1,31 +1,44 @@
 dofile_once("mods/noita-together/files/store.lua")
 dofile_once("mods/noita-together/files/scripts/json.lua")
+-- NT_ban_perks --
+dofile_once("mods/noita-together/files/scripts/ban_perks.lua")
 
 local _item_pickup = item_pickup
 
-function item_pickup( entity_item, entity_who_picked, item_name )
+function item_pickup(entity_item, entity_who_picked, item_name)
+    GamePrint("started item_pickup")
```

Comment:
some logging was not removed. Should only be the last line, and not the first 2
![image](https://github.com/user-attachments/assets/b2ba8d7b-6879-444f-97a0-ffa31a62da65)